### ISSUE TITLE
[Web] [Accessibility][11yMAS][Screen Readers][MAS4.2.10] [Page gets scrolled up when the Voice over focus is on the frame.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#699]
**User Experience:**
User with partial vision disability may get confuse as the focus reaches the frame area , then complete page gets scrolled and user cannot view the frame and current focused element on the frame , but only can listen to the screen reader.

**Test Environment:**
OS Build: Mac OS Sierra 0.12.6
Browser: Safari 11.0.1
Tools Used: Voice Over

**Attachments:**
[MAS4_2_10_PageScrollsUp.zip](https://github.com/OfficeDev/script-lab/files/1456497/MAS4_2_10_PageScrollsUp.zip)

**Repro Steps:**
1. Open the URL: http://scriptlabdocsdemo.azurewebsites.net/Office%20Dev%20Center%20-%20Docs%20-%20Build%20your%20first%20Excel%20add-in.html
2. Navigate to frame using CTRL+ Option +Arrow keys.
3. Press Ctrl+ Option + shift + Down arrow to navigate inside the frame.
4. Verify that the page gets scrolled up as soon as the Voice over focus is inside the frame area.

 **Current Behavior:** 
When user navigate to frame area using Voice over the complete page gets scrolled up.

**Expected Behavior:**
As soon as the focus move inside the frame are , the page should not get scrolled up instead screen reader focus should be visible on the frame to the user. So that user may get the better understanding of the current focused element on the frame.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b24201d1-2276-4471-8183-e1e5353031ad}

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=4.2.10
